src/github.com/getlantern/flashlight/main/pac.go

```diff
@@ -118,42 +131,31 @@ func watchDirectAddrs() {
 	go func() {
 		for {
 			addr := <-detour.DirectAddrCh
-			// prevents Lantern from accidently leave pac on after exits
-			if atomic.LoadInt32(&isPacOn) == 0 {
-				return
-			}
```

Comment:
@myleshorton @fffw FYI, this is a bug that we introduced when we added the ability to disable Lantern as system proxy.  In that case, isPacOn will be false, so we ignore the new direct host.  This is an edge case and unlikely to happen, but I went ahead and fixed it while I was in here.  Specifically, I now check isPacOn only to determine whether to force the browser to reload the PAC by turning it off and then on.  We always apply the direct host to the map of direct hosts.